lib/bundles/inspec-compliance/api.rb

```diff
@@ -238,5 +241,13 @@ def self.server_version_from_config(config)
       return nil unless config['version'].is_a?(Hash)
       config['version']['version']
     end
+
+    def self.determine_server_type(url, insecure)
+      if Compliance::HTTP.get(url + '/compliance/version', nil, insecure).code == '401'
```

Comment:
Could this also return 200 if the user is already logged in?

Reply:
I'm not passing any headers to the `GET` request so it should always act as if the user is not logged in.